adagrams/game.py

```diff
@@ -1,11 +1,273 @@
+import random
+
 def draw_letters():
-    pass
+    
+    letter_pool_str= "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
+    # List comp for converting ABC string on line 7 into list 
+    letter_pool_list = [letter for letter in letter_pool_str]
+    
+    # Original letter pool dict - stays constant
+    letter_pool_dict = {
+        'A': 9, 
+        'B': 2, 
+        'C': 2, 
+        'D': 4, 
+        'E': 12, 
+        'F': 2, 
+        'G': 3, 
+        'H': 2, 
+        'I': 9, 
+        'J': 1, 
+        'K': 1, 
+        'L': 4, 
+        'M': 2, 
+        'N': 6, 
+        'O': 8, 
+        'P': 2, 
+        'Q': 1, 
+        'R': 6, 
+        'S': 4, 
+        'T': 6, 
+        'U': 4, 
+        'V': 2, 
+        'W': 2, 
+        'X': 1, 
+        'Y': 2, 
+        'Z': 1
+    }
+    # Tracking container: where the letters(key) and their freq (value) will be stored as separate dictionaries
+    tracking_letter_dict = {
```

Comment:
Instead of hardcoding a copy of a dictionary, next time try using this:

```python
tracking_letter_dict = letter_pool_dict.copy()
```

Though, I cannot see the advantages of making a copy here. Being that you initialized `letter_pool_dict` inside of the function, each function call it will initialized in it's original state. Therefore, you could directly modify the original dictionary. However, there are a lot of times when you want to use a copy of a mutable object and so I love to see that you are thinking about it!